conductor-client/src/main/kotlin/com/openlattice/assembler/PostgresRoles.kt

```diff
@@ -29,8 +59,8 @@ class PostgresRoles private constructor() {
         }
 
         @JvmStatic
-        fun buildOrganizationRoleName(organizationDbName: String): String {
-            return "${organizationDbName}_role"
+        fun buildOrganizationRoleName(orgId: UUID): String {
+            return buildPostgresRoleName(orgId)
```

Comment:
Is this still used? this change would probably require a migration, although I kind of suspect the organization user role isn't actually doing anything at this point